packages/plugins/c-sharp/c-sharp-operations/src/visitor.ts

```diff
@@ -126,6 +138,26 @@ export class CSharpOperationsVisitor extends ClientSideBaseVisitor<
     return this.config.documentMode === DocumentMode.external ? `Operations.${node.name.value}` : documentVariableName;
   }
 
+  private _gqlInputSignature(variable: VariableDefinitionNode): { signature: string; required: boolean } {
+    const typeNode = variable.type;
+    const innerType = getBaseTypeNode(typeNode);
+    const schemaType = this._schema.getType(innerType.name.value);
+
+    const name = variable.variable.name.value;
+    const baseType = !isScalarType(schemaType) ? innerType.name.value : this.scalars[schemaType.name] || 'object';
+
+    const isArray =
+      typeNode.kind === Kind.LIST_TYPE ||
```

Comment:
Would this work with `field: [[[String]]]`? 🤔

Reply:
Never mind, it's just for indication.

Reply:
Good question. I will check that later and do another pull request if changes are needed.